tools/cli/go-whisk-cli/commands/activation.go

```diff
@@ -315,7 +315,7 @@ var activationPollCmd = &cobra.Command{
                 Limit: 1,
                 Docs:  true,
             }
-            activationList, _, err := client.Activations.List(options)
+            activationList, _, err := Client.Activations.List(options)
```

Comment:
Debug message below should be Client.Activations.List instead of client.Activations.List. Should also display options as the argument in the debug message.